README.md

```diff
@@ -90,46 +66,29 @@ Finally, test to confirm that everything worked correctly! Run the following:
 1. Activate the virual environment: ```. .venv/bin/activate```
 2. Run the flow:  ```python scripts/run_design.py designs/basic/add4/ xilinx_conformal_impl```
 
-## The Ninja Transition
-This branch serves as the transition point to a new version of bfasst, which makes the following changes:
+## Project Structure
+The project makes extensive use of the ninja_build_tool and chevron python package to accomplish the following:
 * It takes advantage of the ninja build tool to automatically handle job creation, up-to-date checking of dependencies, parallel processing, and io redirection. 
 * It uses chevron, the python implementation of the mustache templating engine to template the scripts used for common operations such as synthesis and implementation of designs.
-* It keeps python as its base language, rather than switching to bash in order to minimize the learning curve and maximize portability in transitioning to the new architecture.
-* It provides unit tests for all new code, that can be run with `python -m unittest`. These largely serve as a sanity check for new developers on the project, and should help to maintain minimum requirements around the implementation of new flows.
+* It provides unit tests that can be run with `python -m unittest`. These largely serve as a sanity check for new developers on the project, and should help to maintain minimum requirements around the implementation of new flows.
 
 ### Usage:
+There are two steps to running any flow with any design(s): a ninja generation step and a run step. For convenience, `scripts/run.py` can be used to execute both steps at once. Alternatively, you can execute the ninja generation step entirely separate from the run step for one or more designs using `python bfasst/flows/ninja_flow_manager.py`:
```

Comment:
Can we move this to the project_structure.md file?

Reply:
Done